src/gui/updater/ocupdater.cpp

```diff
@@ -153,12 +153,22 @@ QString OCUpdater::statusString() const
         return tr("Downloading %1. Please wait …").arg(updateVersion);
     case DownloadComplete:
         return tr("%1 available. Restart application to start the update.").arg(updateVersion);
-    case DownloadFailed:
-        return tr("Could not download update. Please open %1 to download the update manually.").arg(_updateInfo.web());
+    case DownloadFailed: {
+        if (format == UpdateStatusStringFormat::Html) {
+            return tr("Could not download update. Please open <a href='%1'>%1</a> to download the update manually.").arg(_updateInfo.web());
+        } else {
```

Comment:
That else statement could be avoided and would it make easier to read in my opinion.